octobot_trading/exchanges/data/exchange_config_data.py

```diff
@@ -43,6 +43,9 @@ def set_config_traded_pairs(self):  # TODO
     def set_config_time_frame(self):  # TODO
         self._set_config_time_frame()
 
+    def get_shortest_time_frame(self):
+        return self.traded_time_frames[-1]
```

Comment:
I don't understand how you ensure that `traded_time_frames[-1]` is the shorted ?

Reply:
Time frames are sorted before being added in exchange config data. I added this method in case this changes, we just have to update this part.

Reply:
Perfect :+1: